sklearn/metrics/tests/test_classification.py

```diff
@@ -160,7 +144,6 @@ def test_precision_recall_f1_score_binary():
                             (1 + 2 ** 2) * ps * rs / (2 ** 2 * ps + rs), 2)
 
 
-@ignore_warnings
```

Comment:
This was unrelated sorry! (I had removed since it didn't raise any warnings)... do I revert?